src/app/helpers/property.js

```diff
@@ -43,6 +43,23 @@ export const getValue = (property) => {
     return value;
 };
 
+/**
+ * Returns true if a property has an empty value
+ * @param {Object} property     { value, field, type, ... }
+ * @return {Boolean}
+ */
+export const isEmptyValued = (property) => {
+    const { value } = property;
+    // property values must be strings or arrays of strings
+    if (typeof value === 'string') {
+        return !value;
+    }
+    if (Array.isArray(value)) {
+        return !value.reduce((acc, str) => acc + str, '');
```

Comment:
Why not using a some ?

Reply:
yeah, that makes sense. The arrays here have length 7 at max so I didn't think of using `some`, but it will be indeed more clear.